backend/src/main/java/ca/bc/gov/hlth/mohums/controller/UsersController.java

```diff
@@ -59,7 +59,7 @@ public UsersController(WebClientService webClientService, @Value("${config.vanit
         return webClientService.get(path, null);
     }
 
-    @PostMapping(value = "/users")
+    @PostMapping( "/users")
```

Comment:
...extra space here.

Reply:
I don't recall asking you to review this.